src/lint/rules/spacing.rs

```diff
@@ -501,3 +753,131 @@ impl Rule for NspTrailingRule {
         RuleType::NspTrailing
     }
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct SpPtrDeclOptions {}
+
+pub struct SpPtrDeclRule {
+    pub enabled: bool,
+}
+
+pub struct SpPtrDeclArgs {
+    type_name_range: ZeroRange,
+    operator_ranges: Vec<ZeroRange>
+}
+
+fn extract_operator_ranges_from_cdecl(node: &CDeclContent) -> Vec<ZeroRange> {
+    node.modifiers.iter()
+            .filter_map(|m| {
+                match m {
+                    LeafToken::Actual(token) => {
+                        match token.kind {
+                            TokenKind::Multiply => Some(m.range()),
```

Comment:
Unless there will be other operators targeted in the future, you can simply this as a
`if token.kind == TokenKind::Multiply` and similarly for the leaftoken you can use `if let LeafToken::Actual(token) = m`